blogs/2016/10/31/js_roundup_2.md

```diff
@@ -0,0 +1,133 @@
+
+Prefer to debug using a different browser? You can find extensions for [Safari](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-ios-web), [Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge), and [Firefox](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-firefox-debug) as well. 
+
+![debugger for chrome image](2016_10_31_chrome_debugger.png)
+
+## Git Project Manager
+
+Marketplace - [Git Project Manager](https://marketplace.visualstudio.com/items?itemName=felipecaputo.git-project-manager)
+
+Publisher - [Felipe Caputo](https://marketplace.visualstudio.com/search?term=publisher%3A%22Felipe%20Caputo%22&target=VSCode&sortBy=Relevance)
+
+Although not necessarily JavaScript specific, this is a favorite for many VS Code team members. This extension scans a directory (or directories) for git repos and allows you to switch between them easily. 
+
+To use this extension, first install it and then add the following code to your `settings.json` file. 
+
+```json
+"gitProjectManager.baseProjectsFolders": [
+    "/path/to/your/base/project/folders"
+]
+```
+
+![git project manager showcase](2016_10_31_git_project_manager.gif)
+
+## Document This
+
+Marketplace - [Document This](https://marketplace.visualstudio.com/items?itemName=joelday.docthis)
+
+Publisher - [Joel Day](https://marketplace.visualstudio.com/search?term=publisher%3A%22Joel%20Day%22&target=VSCode)
+
+Joel Day has built an exceptionally useful extension for writing JavaScript and TypeScript docs. This extension will automatically generates detailed JSDoc comments for your code. 
+
+![document this](2016_10_31_document_this.gif)
+
+## Beautify 
+
+Marketplace - [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify)
+
+Publisher - [HookyQR](https://marketplace.visualstudio.com/search?term=publisher%3A%22HookyQR%22&target=VSCode)
+
+Internally, VS Code uses [js-beautify](https://www.npmjs.com/package/js-beautify). This extension allows you to specify a `.jsbeautifyrc` file to indicate the formatting style for your JavaScript, CSS, Sass, and HTML code. 
+
+You can search the Marketplace for more formatters using the new [Formatters category](https://marketplace.visualstudio.com/search?target=VSCode&category=Formatters&sortBy=Downloads). 
+
+>**Note:** For extension authors, we are working on a blog post for Formatting best practices. Stay tuned as it will be coming soon. 
+
+# Keymaps for Sublime Text and Atom
+
+There are two extensions in this section, one for [Atom](https://marketplace.visualstudio.com/items?itemName=ms-vscode.atom-keybindings) and another for [Sublime Text](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings). If you have used these editors you have likely developed habits with your keyboards shortcuts. These extensions bring the keyboard shortcuts from Atom and Sublime Text to VS Code. 
```

Comment:
Can we just say "Sublime" on the later uses of "Sublime Text". Sublime Text is cumbersome and most people just say "Sublime"